.github/scripts/check_meilisearch_availability.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+PUBLIC_IP=$1
+status="$(curl http://$PUBLIC_IP/health)"
+
+# Check if the variable content is equal to '{"status":"available"}'
+if [ "$status" != '{"status":"available"}' ]; then
+  # If not, echo the variable value and an error message, and exit with code 1
+  echo "Error: Meilisearch is not running correctly."
+  echo "Server response: $status"
+  exit 1
+else
+  # If it is equal, echo a success message and exit with code 0
```

Comment:
I know it's better to comment on your code, but I don't think comments here do much.